plugins/main/public/components/endpoints-summary/table/agents-table.test.tsx

```diff
@@ -429,6 +437,16 @@ describe('AgentsTable component', () => {
     const tableId = '__table_d203a723-1198-11ee-ab9b-75fc624fc672';
     wrapper.find('table')[0]['attribs']['id'] = tableId;
 
+    // Set switch id to avoid snapshot changes
+    const switchButtonId =
+      '_show_outdated_switch_button_ic14ef503-e547-11ee-ac6e-71376e7e3fc5';
+    wrapper.find('.euiSwitch__button')[0]['attribs']['aria-labelledby'] =
+      switchButtonId;
+    wrapper.find('.euiSwitch__button')[0]['attribs']['id'] = switchButtonId;
+    const switchLabelId =
+      '_show_outdated_switch_button_ic14ef503-e547-11ee-ac6e-71376e7e3fc5';
+    wrapper.find('.euiSwitch__label')[0]['attribs']['id'] = switchLabelId;
```

Comment:
**suggestion**: maybe this could be done by mocking the service that generates the ID instead.